README.Rmd

```diff
@@ -27,162 +29,58 @@ options(max.print = 200)
 [![Coverage status](https://codecov.io/gh/r-hub/pkgsearch/branch/master/graph/badge.svg)](https://codecov.io/github/r-hub/pkgsearch?branch=master)
 <!-- badges: end -->
 
-The pkgsearch package searches all CRAN packages. It uses a web service,
-and a careful weighting that ranks popular packages before less
-frequently used ones. The package also has functions to list the new / most recently updated / archived packages, and list all versions of a package.
-
-In this README, read about how `pkgsearch` supports
-
-* [search over all CRAN packages](#search-cran-packages-with-pkgsearch), e.g. by keyword ("colour", "GLM", etc.);
-
-* [extraction of CRAN information](#get-cran-metadata), e.g. releases and archivals.
-
-
-
-## Installation
-
-You can install the package from CRAN:
-
-```r
-install.packages("pkgsearch")
-```
-
-## Search CRAN packages with pkgsearch
-
-For search of CRAN packages, pkgsearch uses a [web service](https://github.com/metacran/search),
-and a careful weighting that ranks popular packages before less
-frequently used ones.
-
-Use `pkg_search()` to search for a term or phrase:
-
-```{r}
-library(pkgsearch)
-pkg_search("C++")
-```
-
-By default it returns a short summary of the ten best search hits. Their
-details can be printed by using the `format = "long"` option of `pkg_search()`,
-or just calling `pkg_search()` again, without any arguments, after a search:
-
-
-```{r}
-pkg_search()
-```
-
-The `more()` function can be used to display the next batch of search hits,
-batches contain ten packages by default. `ps()` is a shorter alias to
-`pkg_search()`:
-
-```{r}
-ps("google")
-```
-
-```{r}
-more()
-```
-
-## Features
-
-In addition to simple text matching the search server and client have
-many improvements to deliver better results.
-
-### Stemming
-
-The search server uses the stems of the words in the indexed metadata, and
-the search phrase. This means that "colour" and "colours" deliver the
-exact same result. So do "coloring", "colored", etc. (Unless one is happen
-to be an exact package name or match another non-stemmed field.)
-
-```{r}
-ps("colour", size = 3)
-ps("colours", size = 3)
-```
+`pkgsearch` uses three R-hub web services ([crandb](https://github.com/r-hub/crandb), [cranlogs](https://github.com/r-hub/cranlogs.app) and [search](https://github.com/metacran/search)) that munge CRAN metadata and let you access it through several lenses.
```

Comment:
I want to be able to change the web service backend, so I would go into the details here.